source/libmongoc/guides/bulk.txt

```diff
@@ -229,9 +236,40 @@ The ``reply`` document is empty:
 
    { }
 
+.. _bulkwrite_server:
+
+``bulkWrite`` Server Command
+----------------------------
+
+The previous examples on this page use the `mongoc_bulk_operation_t <{+api-libmongoc+}/mongoc_bulk_operation_t.html>`__
+type to perform bulk write operations. ``mongoc_bulk_operation_t`` runs the ``insert``, ``update`` and ``delete`` server
+commands. Bulk write operations that use the ``mongoc_bulk_operation_t`` type group operation types and sends them in
```

Comment:
S: I think this can be cut down a bit to make it easier to understand
```suggestion
commands. Bulk write operations that use ``mongoc_bulk_operation_t`` group operations by type and sends them in
```